Header header
int32 robot_id
bool is_ready